packages/nimble-components/src/table-column/date-text/types.ts

```diff
@@ -0,0 +1,107 @@
+/**
```

Comment:
Can you write some simple tests for these types following the patterns we use elsewhere in nimble?